highlight.io/pages/docs/[[...doc]].tsx

```diff
@@ -1111,12 +1106,19 @@ const DocPage = ({
 																						i
 																					}
 																				>
-																					{c?.props?.children?.map(
-																						(
-																							e: any,
-																						) =>
-																							e,
-																					)}
+																					{c
+																						.props
+																						.children
+																						.map
+																						? c?.props?.children?.map(
```

Comment:
Nit, but should the safe navigation (`?.`) be swapped here so it's part of the condition? At this point I think we've already validated that `c.props.children.map` is present.

Reply:
Good catch. I'll fix that on my next PR.